src/pages/questionPage.js

```diff
@@ -40,7 +39,19 @@ export const initQuestionPage = () => {
 };
 
 const nextQuestion = () => {
+  const correctAnswer = quizData.questions[quizData.currentQuestionIndex].correct;
```

Comment:
Very nice! Can you make sure that the 'next question' button can't be clicked twice? Currently it is possible to click the button again during the setTimeout, and then there is some strange behaviour